### PR TITLE
Added field related.user

### PR DESCRIPTION
**Pull request by janniten**
From discussions https://github.com/elastic/ecs/issues/589 and https://github.com/elastic/ecs/issues/678 and from working with windows events https://github.com/elastic/beats/pull/13530 in the winlogbeat security module many events has related users.
In spite of what naming convention/schema will be chosen to represent n-ary user relationships it would be useful to have the related.user field in order to facilitate pivoting.

### CODE REVIEW

File: schemas/related.yml [hunk 1]
```diff
@@ -22,3 +22,9 @@
       type: ip
       description: >
         All of the IPs seen on your event.
+
+    - name: user
+      level: extended
+      type: keyword
+      description: >
+        All of the users seen on your event.
```
Comment (webmat) [inline schemas/related.yml]: I think we should clarify here that it's about user names.

**Comment by webmat** [comment]
Thanks for opening this @janniten!

In making the adjustment above, please make sure to merge master. I think you started from an old version of the master branch. For example, the `schema.json` file should no longer be there, it was removed a little while ago :-)

**Comment by janniten** [comment]
@webmat 
My fork was out of date and I wasn´t able to merge with the master. (probably due to my poor knowledge of git )
I have created a new PR, sorry :(

**Comment by webmat** [comment]
No worries, I can review #694 just as easily :-)

**Pull request closed without merging**